pkg/job_controller/job_controller.go

```diff
@@ -46,6 +46,9 @@ type JobControllerConfiguration struct {
 
 	// Name of global default gang scheduler.
 	GangSchedulerName string
+
+	// The Kaniko image name
+	KanikoImage string
```

Comment:
should we rename this field to a more generic name?  In this way we are able to assign with other image-building-tools but not only `kaniko`.

Reply:
fixed, this config is not used yet, it'll be used in followup issue